### ISSUE TITLE
[Enhancement] Configurable default statuses and labels for new projects

### ISSUE THREAD

**Issue by amconners** [olivierkes/manuskript#1018]
I find myself using the same set of labels for nearly every project (and often the same set of statuses as well), but I don't use the default list. It would be convenient to be able to configure a set of labels and statuses once and have new projects use that setting instead of having to make the same changes every time I create a new project.

(If there's a way to do this and I just missed it please let me know.)

**Comment by TheJackiMonster** [comment]
I think the best you can do currently is after creating a new project (if you do not use the single-file mode), you can close Manuskript and go to the project directory of your older project. Select the "labels.txt" and "status.txt" inside, copy both files and navigate into the project directory of the newer project. Then paste both files there replacing the generated defaults.

This should do the trick to reuse labels and statuses. But I agree that it would be much better to edit the defaults or have some kind of template for projects including such changes.

**Comment by amconners** [comment]
Thanks! I do use single-file mode, but I was able to do the same thing by opening both projects as zip archives and copying the text files between them, which seems to work just fine.

**Comment by TheJackiMonster** [comment]
Yes, that's also possible. Just keep in mind that using the single-file mode is not recommended because there can still occur issues with saving/loading projects and if it only stores one file, it can invalidate everything. Anyway it is recommended to make backups, just to be sure.
